thoth-api/src/contribution/crud.rs

```diff
@@ -152,13 +153,19 @@ impl Crud for Contribution {
         }
     }
 
+    fn publisher_id(&self, db: &crate::db::PgPool) -> Uuid {
```

Comment:
I'm wondering if we should return an `Option<Uuid>` or a `ThothResult<Uuid>` to avoid `unwrap`ing. That'll also allow us replacing the `unimplemented` panics in `Funder` and `Contributor`